common/src/main/resources/application.common.properties

```diff
@@ -2,4 +2,5 @@ efs.mount=${AB2D_EFS_MOUNT:#{systemProperties['java.io.tmpdir']+'/jobdownloads'}
 
 audit.files.ttl.hours=24
 
-hpms.base.url=${AB2D_HPMS_URL:#{'https://hpmsimpl.cms.gov'}}
\ No newline at end of file
+hpms.base.url=${AB2D_HPMS_URL:#{'https://hpmsimpl.cms.gov'}}
+hpms.auth.url=${AB2D_HPMS_AUTH_URL:#{'https://hpmsimpl.cms.gov/api/idm/oauth/token?ACS=wlVuThEThipRlBu37Pra'}}
```

Comment:
This is in both property files, does the HPMS not inherit from common? I'm not sure common is being used though?

Reply:
Cleaned this up.